cmd/download.go

```diff
@@ -46,21 +57,26 @@ func init() {
 	downloadCmd.Flags().StringP("release", "r", "", "OpenShift release version")
 	downloadCmd.MarkFlagRequired("folder")
 	downloadCmd.Flags().StringP("rootfs-url", "u", "", "rootFS URL")
-	downloadCmd.Flags().StringP("ai-installer-sha", "i", "", "AI Installer Image SHA")
-	downloadCmd.Flags().StringP("ai-agent-sha", "g", "", "AI Agent Image SHA")
-	downloadCmd.Flags().StringP("ai-controller-sha", "c", "", "AI Controller Image SHA")
+	downloadCmd.Flags().StringP("ai-installer-sha", "i", "", "AI Installer Image SHA (deprecated)")
+	downloadCmd.Flags().StringP("ai-agent-sha", "g", "", "AI Agent Image SHA (deprecated)")
```

Comment:
I do not think setting the SHA's value to deprecated makes sense since it is not being used by any user. I think that giving the option to the full name is enough for each controller, installer and agent. Actually, it will be similar effort for the user to check the SHA or the full location.

I would move everything to full image location

Reply:
Done